### PR TITLE
Landscape Wayfinding

### LINKED ISSUE #1856: landscape: surface always-available support/wayfinding mechanisms
This issue illustrates a few updates we can make to our support interface(s) to provide better and easier to find wayfinding and onboarding for landscape and the Tlon apps it contains.

In a nutshell, the root of these improvements are staked on surfacing an ever-present "Support Bubble" on all interfaces in the lower left hand corner:

Placement

As shown in the sketches above, this menu is expected to contain a few actions:

- Help and Support
- Submit Feedback
- [NEW] Wayfinding

The "Wayfinding" action in this list corresponds to a new popup interface that displays under these conditions:

1. When this item in the list is explicitly clicked
2. On the first-time access of any given Tlon app.

The rough/overall structure of this wayfinding interface, for now, is simply static text and references to groups/apps at its most complex. Further dynamism can be added to this interface if it's deemed necessary:

General Wayfinding Card Structure

This interface will be tweaked in the future to grant it the ability to be dismissed or conditionally displayed, but for now, let's assume it's very simple.

An overview of this interface, and the copy/wayfinding that might be written to it, can be viewed here, or in [figma](https://www.figma.com/file/xDGJBu9KIPRQqX0KD3kWWJ/2023-Q1-Onboarding?node-id=478%3A26557&t=JtlptLysZTpWyA7w-1) as usual:

Wayfinding Card

### PR DESCRIPTION
**Pull request by patosullivan**
Fixes #1856 for groups and talk. Separate PR incoming for Landscape.

image
image
image

Note that the "reject" buttons on the Group Reference components are a bug, unrelated to this work.

### CODE REVIEW

**Comment by patosullivan** [comment]
@urcades what do you think about adding an option to hide the question mark icon/menu?

**Comment by urcades** [comment]
@patosullivan I'm into it — we'd simply need to surface a means to bring it back up + reset its state if need be — we have the "Help & Support" section of Landscape's system preferences, so a toggle could potentially live here.

**Comment by patosullivan** [comment]
> @patosullivan I'm into it — we'd simply need to surface a means to bring it back up + reset its state if need be — we have the "Help & Support" section of Landscape's system preferences, so a toggle could potentially live here.

Cool. I'll add a "Hide This Button" option here and then add a toggle in the Landscape system preferences.

**Comment by patosullivan** [comment]
> 🫡
> 
> we covered the placement of the toggle for resetting the state of the help bubble

Hey, just realized this is going to look weird on mobile: 

localhost_3000_apps_groups_(Pixel 5) (1)

Should we hide it on mobile?

**Comment by urcades** [comment]
@patosullivan in the previous issue where these were first posted, it was recommended that we float this button slightly above the toolbar:

Screenshot 2023-02-07 at 12 52 41 PM

This is still quite in the way imo, and will need some fine-tuning to feel good. I think we'll be able to address this guy in a more ideal/out of the way manner when we start working on system chrome/ui that links everything together

**Comment by patosullivan** [comment]
> @patosullivan in the previous issue where these were first posted, it was recommended that we float this button slightly above the toolbar:
> 
> Screenshot 2023-02-07 at 12 52 41 PM
> 
> This is still quite in the way imo, and will need some fine-tuning to feel good. I think we'll be able to address this guy in a more ideal/out of the way manner when we start working on system chrome/ui that links everything together

Ok, I'll bump it up.

**Comment by patosullivan** [comment]
status: bumped

localhost_3000_apps_groups_(Pixel 5) (3)

**Comment by urcades** [comment]
Nice, this is looking good

File: ui/src/components/LandscapeWayfinding.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+  return (
+    <Popover.Root>
+      <div
+        className={cn('absolute left-5 z-50', {
+          'bottom-10': !isMobile,
+          'bottom-20': isMobile,
+        })}
+      >
+        <Popover.Trigger className="relative" asChild>
```
Comment (arthyn) [inline ui/src/components/LandscapeWayfinding.tsx:102]: Any reason why popover instead of dropdown menu here?

**Review by arthyn — APPROVED** [review]
lgtm, just the one question

File: ui/src/components/LandscapeWayfinding.tsx (hunk 1 above)
Comment (patosullivan) [reply ui/src/components/LandscapeWayfinding.tsx:102]: No, if you'd prefer we can switch to dropdown menu. I guess I just mentally categorize this as a "popover".

**Pull request merged**